### PR TITLE
fix(test): Remove extra `v2/schema` import

### PR DESCRIPTION
**Pull request by candiduslynx**

### CODE REVIEW

File: schema/testdata.go [hunk 1]
```diff
@@ -361,7 +360,7 @@ func getExampleJSON(colName string, dataType arrow.DataType, opts GenTestDataOpt
 	if arrow.IsListLike(dataType.ID()) {
 		if dataType.ID() == arrow.MAP {
 			k := getExampleJSON(colName, dataType.(*arrow.MapType).KeyType(), opts)
-			v := getExampleJSON(colName, dataType.(*arrow.MapType).ValueType().Field(1).Type, opts)
+			v := getExampleJSON(colName, dataType.(*arrow.MapType).ItemType(), opts)
```
Comment (hermanschaaf) [inline schema/testdata.go:363]: This didn't work when I tried it initially, but maybe it works now? Has the implementation changed? Have you tested it?

Comment (hermanschaaf) [reply schema/testdata.go:363]: Oh no, I'm probably misremembering: I think it was that `ValueType()` didn't return what I expected in the same way as `KeyType()`, so yeah maybe `ItemType()` is the way to go. I'll need to check the implementation though

Comment (hermanschaaf) [reply schema/testdata.go:363]: Yeah looks fine... bit confusing naming, but anyway :)

**Review by hermanschaaf — APPROVED** [review]

File: schema/testdata.go (hunk 1 above)
Comment (candiduslynx) [reply schema/testdata.go:363]: Yeah, I had to dig a bit during CH implementation for it.
Basically, `Map(keyType, itemType)` is `List(Struct("key" keyType, "value" nullable(valueType)))` in Arrow.
Then we have 3 different sets of functions:
* `KeyType` & `KeyField` for key element
* `ItemType` & `ItemField` for value element
* `ValueType` & `ValueField` for `list element` (as map = list(struct(...)))

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**